test2.py

```diff
@@ -0,0 +1,39 @@
+# from pieces import *
+# from board import *
+# from ascii import *
+
+# board = Board()
+# board.new_game()
+
+# board.update_moves()
+
+# board.ascii.draw_board(board)
+
+
+# board.squares[35].piece = board.squares[33].piece
+# board.squares[33].piece = None
+# board.update_moves()
+
+# board.ascii.draw_board(board)
+
+# print
+
+# a = "move e7 to e5"
+
+# algebraic_squares = set()
+
+# columns = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
+# rows = range(1,9)
+# for column in columns:
+# 	for row in rows:
+# 		algebraic_square = column + str(row)
+# 		algebraic_squares.add(algebraic_square)
+
+
+# if a.startswith("move "):
+# 	print(a[5:7])
+# 	print (a[11:13])
+x = 0
+while x < 10:
+	counter += 1
+	x += 1 
```

Comment:
I think there is nothing useful or worth reviewing in this file. So if it's something in progress consider keeping it locally but not including it in commits.

Reply:
In general, you will also want to think about where to put tests (some people put in a separate folder, others in the same folder and named after the file that is under test) and make sure to name them useful things! 

`test.py` and `test2.py` aren't super helpful. But if you had `board_test.py` and `ascii_test.py`, it would make it clearer what is under test in each.

Reply:
This should have been in my gitignore. Apologies.